README.md

```diff
@@ -10,4 +10,61 @@ I will not be guarenting anything. I'll try to target both bluesky & mastodon, b
 
 ### This is not (and probably won't be for a while) a 100% accurate recreation of the twitter API
 
-most of this readme todo
\ No newline at end of file
+most of this readme todo
+
+## Compatibility
+**iOS:** 4.2.1 works
+
+**Android:** Untested
+
+**Windows Phone 8/10:** Untested 
+
+**BlackBerry:** Untested
+
+**Nokia Series 40:** Untested
+
+**Windows 8 Metro:** Untested
+
+**Windows 10 UWP:** Untested, lost media?
+
+**macOS:** Untested, lost media
+
+**Tizen Mobile:** Untested, unconfirmed existance
+
+**Apple Watch:** Untested
+
+**Samsung Orsay:** Untested
+
+**Tizen TV:** Untested
+
+**LG NetCast:** Untested
+
+**LG webOS:** Untested, lost
+
+**Yahoo! Connected TV:** Untested, lost
+
+**Xbox 360:** Untested
+
+**Xbox One:** Untested, lost
+
+**Apple TV:** Untested
+
+**Android TV:** Untested
+
+**PS Vita:** Untested
+
+**Samsung T9000 Fridge:** Untested, undumped
```

Comment:
This is just android pretty sure.

Reply:
That fridge runs Android but the client is specifically built for that fridge.